src/components/modebar/modebar.js

```diff
@@ -175,6 +181,15 @@ proto.createButton = function(config) {
             // only needed for 'hoverClosestGeo' which does not call relayout
             _this.updateActiveButton(ev.currentTarget);
         });
+        button.addEventListener('keydown', function(e) {
```

Comment:
@davidangarita1 we talked about this i don't think this is necessary now that we're using button?